config/sync/migrate_plus.migration.stanford_localist_importer.yml

```diff
@@ -494,6 +494,68 @@ process:
           entity_type: taxonomy_term
           ignore_case: true
           source: name
+  image_ext:
+    - plugin: skip_on_empty
+      method: process
+      source: photo_url
+    - plugin: explode
+      delimiter: .
+    - plugin: array_pop
+  image_destination_part_1:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_ext'
+    - plugin: concat
+      source:
+        - constants/image_path
+        - urlname
+  image_destination:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_destination_part_1'
+    - plugin: concat
+      source:
+        - '@image_destination_part_1'
+        - '@image_ext'
+      delimiter: .
+  image_file:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_destination'
+    - plugin: file_import
+      source: photo_url
+      destination: '@image_destination'
+      id_only: true
+  media_tag:
+    plugin: entity_generate
+    value_key: name
+    bundle_key: vid
+    bundle: media_tags
+    entity_type: taxonomy_term
+    ignore_case: true
+    source: constants/image_tag
```

Comment:
You still need to either define the `image_tag` constant, or use default value. You can see the `constants:` above.
To use `default_value`, it would look like this:
```suggestion
  media_tag:
    -
      plugin: default_value
      default_value: 'Localist Event'
    -
      plugin: entity_generate
      value_key: name
      bundle_key: vid
      bundle: media_tags
      entity_type: taxonomy_term
      ignore_case: true
```

Reply:
Hi Mike. 

I'm reviewing the examples. I'll let you know when it's ready.

Reply:
Thanks for the info.

Reply:
@pookmish I'm trying to figure out if there's way to verify this is working. I presume this plugin should add a tag to the media from a taxonomy term. Is that correct?

Reply:
correct. and the imported content should be tagged with the term.

Reply:
And that's created on the cron run? I'm not seeing anything in Media Tags:

![Uploading Screenshot 2023-11-01 at 12.34.01 PM.png…]()

Reply:
because you still have the `source: constants/image_tag`. You have to use either the default_value or the constant. not both.

Reply:
Hm. I thought I removed that.

Reply:
I see it now. Okay. I think it's ready now.